packages/@react-aria/virtualizer/src/Virtualizer.tsx

```diff
@@ -178,13 +179,18 @@ export function useVirtualizer<T extends object, V, W>(props: VirtualizerOptions
     }
   });
 
+  let hasTabbableChild = useHasTabbableChild(ref);
```

Comment:
can probably do something like this
```suggestion
  let hasTabbableChild = useHasTabbableChild(ref, virtualizer.collection.size === 0, virtualizer.contentSize.height > 0 || virtualizer.contentSize.width > 0);
```